sklearn/neural_network/_rbm.py

```diff
@@ -77,28 +77,23 @@ class BernoulliRBM(TransformerMixin, BaseEstimator):
         Biases of the visible units.
 
     components_ : array-like of shape (n_components, n_features)
-        Weight matrix, where n_features in the number of
+        Weight matrix, where n_features is the number of
         visible units and n_components is the number of hidden units.
 
     h_samples_ : array-like of shape (batch_size, n_components)
         Hidden Activation sampled from the model distribution,
-        where batch_size in the number of examples per minibatch and
+        where batch_size is the number of examples per minibatch and
         n_components is the number of hidden units.
 
     n_features_in_ : int
         Number of features seen during :term:`fit`.
 
         .. versionadded:: 0.24
 
-    Examples
+    See Also
     --------
 
-    >>> import numpy as np
-    >>> from sklearn.neural_network import BernoulliRBM
-    >>> X = np.array([[0, 0, 0], [0, 1, 1], [1, 0, 1], [1, 1, 1]])
-    >>> model = BernoulliRBM(n_components=2)
-    >>> model.fit(X)
-    BernoulliRBM(n_components=2)
+    gibbs: Gibbs sampling step.
```

Comment:
Here we should document some public functions or public classes. `gibbs` is a method of the estimator so it would not fit here. However, I am not sure which public class to document here. @ogrisel Do you think that it makes sense to add any manifold learning method or a PCA?

Reply:
So MLPClassifier/MLPRegressor and PCA are good candidates.